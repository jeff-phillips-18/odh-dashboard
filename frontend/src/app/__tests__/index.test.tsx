import * as React from 'react';
import { App } from '../';
import { shallow } from 'enzyme';

describe('Index test', () => {
  it('should render a basic component', () => {
    const component = shallow(<App />);
    expect(component.html()).toMatchSnapshot('basic');
  });
});
