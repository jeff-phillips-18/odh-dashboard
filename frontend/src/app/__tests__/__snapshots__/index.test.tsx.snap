// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index test should render a basic component: basic 1`] = `"hello world!"`;
